content/en/opentelemetry/setup/collector_exporter/_index.md

```diff
@@ -103,26 +118,31 @@ service:
       exporters: [datadog/exporter]
 ```
 
-The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries over HTTP and gRPC, and sets up a [batch processor][5], which is mandatory for any non-development environment. Note that you may get `413 - Request Entity Too Large` errors if you batch too much telemetry data in the batch processor.
+This basic configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries over HTTP and gRPC, and sets up a [batch processor][5].
+
+For a complete list of configuration options for the Datadog Exporter, see the [fully documented example configuration file][8]. Additional options like `api.site` and `host_metadata` settings may be relevant depending on your deployment.
```

Comment:
Use `::` for nested configuration keys, since `.` is a valid character in a configuration key
```suggestion
For a complete list of configuration options for the Datadog Exporter, see the [fully documented example configuration file][8]. Additional options like `api::site` and `host_metadata` settings may be relevant depending on your deployment.
```